account/src/main/java/org/fiware/tmforum/account/domain/BillingAccount.java

```diff
@@ -12,17 +12,41 @@
 import org.fiware.tmforum.common.domain.Money;
 import org.fiware.tmforum.common.domain.TimePeriod;
 import org.fiware.tmforum.product.CategoryRef;
+import org.fiware.account.model.PaymentMethodRefVO;
 
 import java.net.URI;
 import java.time.Instant;
 import java.util.List;
 
 @EqualsAndHashCode(callSuper = true)
-@MappingEnabled(entityType = BillingAccount.TYPE_PARTYAC)
-public class BillingAccount extends PartyAccount {
+@MappingEnabled(entityType = BillingAccount.TYPE_BILLINGAC)
+public class BillingAccount extends Account {
 
     public BillingAccount(String id) {
-        super(id);
+        super(TYPE_BILLINGAC, id);
     }
 
+    public static final String TYPE_BILLINGAC = "billingAccount";
```

Comment:
I think this type should be billing-account for consistency with the other APIs (is the type used in BillingAccountRef)